jkube-kit/common-maven/src/test/java/org/eclipse/jkube/kit/common/util/MavenUtilTest.java

```diff
@@ -57,110 +57,103 @@
 import org.junit.Test;
 import org.junit.rules.TemporaryFolder;
 
-import static org.hamcrest.CoreMatchers.equalTo;
-import static org.hamcrest.MatcherAssert.assertThat;
-import static org.hamcrest.Matchers.contains;
-import static org.hamcrest.Matchers.hasSize;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.assertNotNull;
+import static org.assertj.core.api.Assertions.assertThat;
 
 public class MavenUtilTest {
-  @Rule
-  public TemporaryFolder temporaryFolder = new TemporaryFolder();
-  @Mocked
-  private KitLogger log;
+    @Rule
+    public TemporaryFolder temporaryFolder = new TemporaryFolder();
+    @Mocked
+    private KitLogger log;
```

Comment:
You seem to have changed indentation now to 4 characters. Please configure your IDE as per project requirements https://www.eclipse.org/jkube/contributing#ide-config-and-code-style

Reply:
Thanks for the resource, will make sure to format in next merge as well. Resolved.